test/e2e/specs/interactivity/directive-context.spec.ts

```diff
@@ -189,4 +189,12 @@ test.describe( 'data-wp-context', () => {
 		await page.getByTestId( 'async navigate' ).click();
 		await expect( element ).toHaveText( 'changed from async action' );
 	} );
+	test( 'should bail out if the context is not a default directive', async ( {
+		page,
+	} ) => {
+		// This test is to ensure that the context directive is only applied to the default directive
+		// and not to any other directive.
+		const element = page.getByTestId( 'non-default suffix context' );
+		await expect( element ).toHaveText( '' );
```

Comment:
I'd include a test before this one to check whether the default context is working with a non-default context. ☝️

Reply:
This does not work, right?
```
<div
	data-wp-interactive='{"namespace": "directive-context-non-default"}'
	data-wp-context--non-default='{ "text": "non default" }'
	data-wp-context='{ "defaultText": "default" }'
	data-wp-context='{ "anotherText": "another" }'
>
	<span data-testid="non-default suffix context" data-wp-text="context.text"></span>
	<span data-testid="default suffix context" data-wp-text="context.defaultText"></span>
	<span data-testid="another suffix context" data-wp-text="context.anotherText"></span>
</div>
```
I think that's the reason I did not added the default text. Cause two contexts are not allowed in one single element I guess.